src/__tests__/App.test.js

```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import React from 'react';
+import store from '../Redux/store';
+import App from '../App';
+
+describe('App js testing', () => {
```

Comment:
- [ ] Your test cases are all failing. Kindly review and fix them.
![image](https://github.com/MenendezON/react-capstone/assets/62849872/76dcf6ed-75a8-4079-a21b-fcff8c27b13b)